### ISSUE TITLE
Can it be used as a SAAS

### ISSUE THREAD

**Issue by ugintl** [arnaudom/ek#3]
Can it be used to make a saas app? I mean users will register and can have their own erp. Sub-domain or sub-site. Something like that.

**Comment by arnaudom** [comment]
Yes, of course; but has it is the modules will not provide the automated Saas account creation and  the required installation for the customer. You will have to add a layer of code for that, a profile or install manually each account on demand.

**Comment by ugintl** [comment]
There is a concept of muti site in drupal. Do you know about it?

**Comment by arnaudom** [comment]
Yes, you can use multi-sites configuration, but you have to automate the setup of multisite, drupal install, server configuration and other settings if you want a system to run from a simple user registration.
